### PR TITLE
Release cassandra salesforce

### PR DESCRIPTION
**Pull request by peshen**
New feature for ADF copy
### Feature Additions
- The following linked service types have been added: 
  - OnPremisesCassandraLinkedService, SalesforceLinkedService
- The following dataset types have been added: 
  - OnPremisesCassandraTableDataset
- The following copy source types have been added:  
  - CassandraSource

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@peshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@peshen If you are internal to Microsoft, please follow the instructions here for joinging the Azur eorg:  http://aka.ms/azuregithub

**Comment by hovsepm** [comment]
@azuresdkci test this please

**Comment by markcowl** [comment]
@peshen no tests?

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@peshen we cannot even look at this PR until the CLA issue is resolved.  Please respond on thsi PR, or we will close it today.

**Comment by jackmagic313** [comment]
As Mark mentioned, please add object samples in azure-sdk-for-net/src/ResourceManagement/DataFactory/DataFactory.Tests/Framework/JsonSamples/ and resend the pull request.

**Comment by peshen** [comment]
Hi mark, I have done http://aka.ms/azuregithub before and have successfully complete a pull request. Now what's wrong with my account?

**Comment by peshen** [comment]
@markcowl

**Comment by markcowl** [comment]
@peshen please add samples.tests per @jackmagic313 's comment.

**Comment by peshen** [comment]
@markcowl I am not sure what "Build finished. No test results found." mean. Even I add some test, there still will be this error.

**Comment by bgold09** [comment]
@peshen you have merge conflicts, that's why no tests ran. Also per the previous comments, you must add a test for these features.

File: src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 For additional details on features, see the full [Azure Data Factory Release Notes](https://azure.microsoft.com/en-us/documentation/articles/data-factory-release-notes). 
 
+## Version 4.10.0
+_Release date: 2016.07.22_
```
Comment (bgold09) [inline src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md]: Please change this date

**Comment by peshen** [comment]
@bgold09 But it shows build finished. Why build can finished with merge conflict?

**Comment by peshen** [comment]
@markcowl  Looks good, finally..

**Comment by peshen** [comment]
@markcowl can we merge?

**Pull request merged**